app/controllers/application_controller/ci_processing.rb

```diff
@@ -954,18 +954,20 @@ def reconfigure_update
       if(params[:id] && params[:id] != 'new')
         @request_id = params[:id]
       end
-      if VmReconfigureRequest.make_request(@request_id, options, current_user)
-        flash = _("VM Reconfigure Request was saved")
-        if role_allows?(:feature => "miq_request_show_list", :any => true)
-          javascript_redirect :controller => 'miq_request', :action => 'show_list', :flash_msg => flash
-        else
-          url = previous_breadcrumb_url.split('/')
-          javascript_redirect :controller => url[1], :action => url[2], :flash_msg => flash
-        end
+
+      ApplicationRecord.group_ids_by_region(params[:objectIds]).each do |region, ids|
```

Comment:
@carbonin - I think VMReconfigureRequest  should handle creating the requests for multiple regions - not this controller.